EIPS/eip-2334.md

```diff
@@ -20,11 +20,11 @@ A standard for allocating keys generated by [EIP-2333](./eip-2333.md) to a speci
 
 ## A note on purpose
 
-This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted the BLS12-381 signature standard. It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
+This specification is designed not only to be an Ethereum 2.0 standard, but one that is adopted by the wider community who have adopted Boneh–Lynn–Shacham (BLS) signatures on the [BLS12-381 curve](https://electriccoin.co/blog/new-snark-curve/). It is therefore important also to consider the needs of the wider industry along with those specific to Ethereum. As a part of these considerations, it is the intention of the author that this standard eventually migrate to a more neutral repository in the future.
```

Comment:
Again, I'd prefer to push readers towards the CFRG draft, if anything.

Reply:
makes sense, it's the authoritative reference that Eth uses, will add it

Reply:
Please see https://github.com/veorq/EIPs/commit/17242295ce1be1ad573ac7b972cf2d7b35d9e737